### ISSUE TITLE
Feature Request: Reordering scenes with children also moves children

### ISSUE THREAD

**Issue by commonroom** [kevboh/longform#161]
**Describe the solution you'd like**

When a scene has children, moving the parent scene should move the children as well.

For example: 

```yaml
...
  scenes:
    - parent scene # moving this one
    - - child 1
    - - child 2 
    - - child 3
    - soon-to-be parent
```

When moving `parent scene` under `soon-to-be parent`, the children `child 1`, `child 2`, and `child 3` should also move so the result is as follows: 

```yaml
...
  scenes:
    - soon-to-be parent
    - - parent scene # all children moved with it
      - - child 1
      - - child 2
      - - child 3
```

**Describe alternatives you've considered**

Selecting multiple scenes using `shift` or `cmd/ctrl` click

**Comment by w0010** [comment]
Yes, please. Re-ordering chapters with many scenes inside is so labor intensive—having to drag them scene by scene—that it's rendered useless for me. Being able to selecting multiple scenes could be a great solution.

**Comment by AndrewMWebster** [comment]
+1 from me.

**Comment by kevboh** [comment]
I agree that this would be a good feature. Unfortunately, implementing it is very difficult—I tried during the initial implementation of v2 and punted. For now, editing the index file directly is your best bet, and is functionally doing the same thing. But trust me that I want this as much as you do, and it's on my radar.

**Comment by vertigomaster** [comment]
@kevboh I might try taking a stab at this myself pretty soon; I've been meaning to get into Obsidian plugins for a while now, so this might be a good excuse for me to actually start, lol

Is there an already existing branch I should look at or any other more recent info I should know? This is an older thread, so I'm not sure where you're at with the feature now
